src/content/reference/react/use-client.md

```diff
@@ -31,20 +31,19 @@ Add `'use client'` at the top of a file to mark the module and its transitive de
 
 import { useState } from 'react';
 import { formatDate } from './formatters';
-import Link from './link';
+import Button from './link';
 
-export default function RichTextEditor(props) {
-  formatDate();
+export default function RichTextEditor({ timestamp, text }) {
+  const date = formatDate(timestamp);
+  // ...
+  const editButton = <Button />;
   // ...
-  return
-    <Link />
-    // ...
```

Comment:
found this syntax awkward. changed to a Button because I couldn't think of a variable name for the link